### PR TITLE
Add translation bind single version in saas singleton 

### PR DESCRIPTION
**Pull request by tigershi**
[#1888](https://github.com/vmware/singleton/issues/1888)

### CODE REVIEW

File: g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/v2/combine/TranslationWithPatternAPI.java [hunk 1]
```diff
@@ -59,9 +59,12 @@ public APIResponseDTO getRranslationWithPattern(
           @ApiParam(name = APIParamName.REGION, required=false, value = APIParamValue.REGION) @RequestParam(value = APIParamName.REGION, required=false, defaultValue="") String region,
           @ApiParam(name = APIParamName.PSEUDO, required=false, value = APIParamValue.PSEUDO) @RequestParam(value = APIParamName.PSEUDO, required=false, defaultValue="false") String pseudo,
           @ApiParam(name = APIParamName.SCOPE_FILTER, required = false, value = APIParamValue.SCOPE_FILTER) @RequestParam(value = APIParamName.SCOPE_FILTER, required = false) String scopeFilter,
-          HttpServletRequest req)  throws Exception  {
-      
-       return super.getTransPattern(combine, productName, version, components, language, scope, region, pseudo, scopeFilter);
-   }
+			HttpServletRequest req) throws Exception {
+		if (req.getAttribute("rebindVersin") != null) {
```
Comment (Xiaochao8) [inline g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/v2/combine/TranslationWithPatternAPI.java:63]: rebindVersin -> rebindVersion

Comment (tigershi) [reply g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/v2/combine/TranslationWithPatternAPI.java:63]: done

**Comment by linr211** [comment]
where's the issue or JIRA task link?

**Review by jin-jasper — DISMISSED** [review]

File: g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/base/utils/VersionMatcher.java [hunk 2]
```diff
@@ -27,7 +27,6 @@ public static String getMatchedVersion(final String version,
             final List<String> vList) throws L3APIException {
```
Comment (linr211) [inline g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/base/utils/VersionMatcher.java:27]: i don't see which codes trigger  throwing  this exception, should remove 'throws L3APIException'?

Comment (tigershi) [reply g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/base/utils/VersionMatcher.java]: done

**Review by linr211 — DISMISSED** [review]

**Review by jin-jasper — APPROVED** [review]

**Review by linr211 — APPROVED** [review]

**Pull request merged**